playground/components/framework-switcher.tsx

```diff
@@ -15,7 +15,7 @@ const frameworks: FrameworkConfig[] = [
 
 export function FrameworkSwitcher() {
   const { dispatch } = useSandpack();
-  const setFramework = useSetAtom(frameworkAtom);
+  const [framework, setFramework] = useAtom(frameworkAtom);
 
   const handleSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
in general, should we call it `framework` or `preset`? like, will be a scenario where we want to have a `next` and `next-ts` presets? in this case it might make more sense calling it PresetSwitcher, and use `preset` on the searchParams instead of `framework`

Reply:
@aidenybai @GuiBibeau

Reply:
I agree preset makes a bit more sense then framework. But this is mostly semantics and not really an issue for this PR. We could make a follow up for it if we want to.